0.data-download/scripts/data_loader.py

```diff
@@ -43,21 +43,22 @@ def load_data(data_directory, adult_or_pediatric = "all"):
        sample_df = sample_df.query("age_categories == @adult_or_pediatric").reset_index(drop=True)
        samples_to_keep = sample_df.reset_index(drop=True).DepMap_ID.tolist()  
        dependency_df = dependency_df.query("DepMap_ID == @samples_to_keep").reset_index(drop=True)
-       
     
     return sample_df, dependency_df
+    
+    sample_df = sample_df.set_index("DepMap_ID").reset_index()
```

Comment:
b/c this line is after the `return` statement, it will never execute. Do you mean to add it before the return statement? If not, please delete the line